src/test/java/io/confluent/connect/jdbc/sink/integration/PostgresDatatypeIT.java

```diff
@@ -85,10 +91,96 @@ public void after() throws SQLException {
     log.info("Dropped table");
   }
 
+  /**
+   * Verifies that even when the connector encounters exceptions that would cause a connection
+   * with an invalid transaction, the connector sends only the errant record to the error
+   * reporter and establishes a valid transaction for subsequent correct records to be sent to
+   * the actual database.
+   *
+   * @throws Exception
+   */
+  @Test
+  public void testPrimaryKeyConstraintsSendsToErrorReporter() throws Exception {
+    props.put(ERRORS_TOLERANCE_CONFIG, ToleranceType.ALL.value());
+    props.put(DLQ_TOPIC_NAME_CONFIG, DLQ_TOPIC_NAME);
+    props.put(DLQ_TOPIC_REPLICATION_FACTOR_CONFIG, "1");
+    props.put(MAX_RETRIES, "2");
```

Comment:
Just curious, what's the purpose of setting this property? My understanding is that, as long as the failures encountered while writing to the database are deterministic, the interactions that the connector makes with the error reporter should be the same regardless of the number of retries it's configured with. Am I missing something? 😅

Reply:
Honestly, it's just to reduce the length of the test, since otherwise we'd have to wait for 10 retries 😆

Reply:
Ah, good idea! In that case, does it make sense to set it to zero? And would the testing time be reduced if we set it elsewhere?

Reply:
@aakashnshah ping :)

Reply:
Yeah , I think it is fair to just put to 0, and yes I will check out other places where this may be applicable.

Reply:
Does it make sense to add this to the view integration test as well?

Reply:
@aakashnshah ping :)
Do you think this makes sense to add anywhere else?